src/App.js

```diff
@@ -44,6 +48,13 @@ class App extends Component {
   getUser = (username, password)  => {
     this.authorizeUser(username, password)
   }
+
+  postUserRating = (userId, movieId, userRating) => {
```

Comment:
Did we ever figure out what Khalid was saying about how we're structuring api calls?